src/ert/ensemble_evaluator/_builder/_ensemble.py

```diff
@@ -161,16 +161,13 @@ def _create_snapshot(self) -> Snapshot:
                 active=True,
                 status=state.REALIZATION_STATE_WAITING,
             )
-            for step in real.steps:
-                reals[str(real.iens)].steps[str(step.id_)] = Step(
-                    status=state.STEP_STATE_UNKNOWN
+            reals[str(real.iens)].step = Step(status=state.STEP_STATE_UNKNOWN)
```

Comment:
What is this step singleton used for? Can we move the status up to realization instead?

Reply:
The (Legacy)Step still contains information that we need in the ensemble_evaluator so it cannot be just ripped out. This object is to be removed in a future PR where it gets merged upwards up to the realizaiton object, yes.